### PR TITLE
move owner into ttx

### PR DESCRIPTION
**Pull request by adecaro**
This PR moves the owner service inside the ttx service. This is in alignment with `audit` and `auditdb` services, and `tokens` and `tokendb` services.

### CODE REVIEW

File: token/services/ttx/manager.go [hunk 1]
```diff
@@ -18,32 +18,32 @@ import (
...
-// NewManager creates a new Owner manager.
-func NewManager(np NetworkProvider, ttxdbManager TTXDBProvider, storage storage.DBEntriesStorage) *Manager {
+// NewManager creates a new DB manager.
+func NewManager(np NetworkProvider, ttxdbManager DBProvider, storage storage.DBEntriesStorage) *Manager {
 	return &Manager{
 		networkProvider: np,
 		storage:         storage,
 		ttxdbProvider:   ttxdbManager,
-		owners:          map[string]*Owner{},
+		owners:          map[string]*DB{},
```
Comment (alexandrosfilios) [inline token/services/ttx/manager.go:41]: Maybe we can rename this field to txDbs or something similar.

File: token/services/ttx/manager.go [hunk 2]
```diff
@@ -65,12 +65,12 @@ func (cm *Manager) Owner(tmsID token.TMSID) (*Owner, error) {
 	return c, nil
 }
 
-func (cm *Manager) newOwner(tmsID token.TMSID) (*Owner, error) {
+func (cm *Manager) newOwner(tmsID token.TMSID) (*DB, error) {
```
Comment (alexandrosfilios) [inline token/services/ttx/manager.go:68]: and this one to newDB.

**Review by alexandrosfilios — APPROVED** [review]
LGTM

File: token/services/ttx/manager.go (hunk 1 above)
Comment (adecaro) [reply token/services/ttx/manager.go]: 👍

**Pull request merged**